src/open_clip/loss.py

```diff
@@ -119,3 +119,48 @@ def forward(self, image_features, text_features, logit_scale):
             F.cross_entropy(logits_per_text, labels)
             ) / 2
         return total_loss
+
+
+class CoCaLoss(ClipLoss):
```

Comment:
not sure if inheritance is the right relationship here, would love some external opinion on this (@rom1504). Feels like it's more like CoCaLoss HAS ClipLoss and not IS a type of ClipLoss. Not super important though

Reply:
I had done like you say with with clip_loss as a CoCaLoss attribute but then I saw I would need to almost rewrite the init so I changed it this, can go back if it is better

Reply:
I agree it would be best to avoid inheritance here